src/Kunstmaan/MediaBundle/Helper/Imagine/BackgroundFilterLoader.php

```diff
@@ -17,22 +19,6 @@ class BackgroundFilterLoader extends \Liip\ImagineBundle\Imagine\Filter\Loader\B
      */
     public function load(ImageInterface $image, array $options = array())
     {
-        $background = new Color(
```

Comment:
Leave this part unchanged as the upmerge will "fix" this. Otherwise it might cause a merge conflict. Only add the deprecation message here on master.